generic/tests/mac_change.py

```diff
@@ -160,7 +160,10 @@ def run(test, params, env):
                                       logging.info)
                 utils_test.run_file_transfer(test, params, env)
         else:
-            check_guest_mac(test, new_mac, vm)
+            if params.get("ctrl_mac_addr") == "off":
+                check_guest_mac(test, old_mac, vm)
```

Comment:
Hi, do you need to update Line114? It will not raise a exception when `ctrl_mac_addr = off`

Reply:
Current test status is:
Linux: ctrl_mac_addr = off -> in guest: new mac set successfully, new ip get normally
                                              'info network': mac still be the old one
          ctrl_mac_addr = on -> in guest and 'info network': mac will changed to the new one

Windows: behave differently from linux guest.

Reply:
Thanks, got it.

Reply:
@xiywang for windows part, it should behave the same as linux guest, but developer are still looking into it. We can modify windows part after windows part fixed.